tests/pytests/unit/runners/test_state.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
```

Comment:
```suggestion

```

Reply:
Good catch, I should turn off the python file template that inserts the shebang